### ISSUE TITLE
Update grub-btrfs version

### ISSUE THREAD

**Issue by geckolinux** [SpiralLinux/SpiralLinux-project#205]
https://github.com/Antynea/grub-btrfs/releases/tag/4.12

**Comment by methanoid** [comment]
Maybe if we get a Spiral update with Debian 12 base we can get this at same time?

**Comment by geckolinux** [comment]
Yep, it has already been applied to the upcoming SpiralLinux configuration.

**Comment by methanoid** [comment]
Upcoming.... Now I am interested.... I might even ask "ETA WEN" ... I dont want to install Spiral 11 and upgrade to 12.. I want to start on 12 :-)
